### ISSUE TITLE
How to debug serverless application which uses serverless framework 

### ISSUE THREAD

**Issue by sumanth712bs** [aws/aws-toolkit-jetbrains#2134]
**Your Environment**

- OS:  MAC OS
- JetBrains product: IntelliJ Idea Ultimate
- JetBrains product version: 2020.2.1
- AWS Toolkit version: 1.18-202
- SAM CLI version:  1.3.2
- JVM/Python version: 11

**Question**

Hey, I recently started using AWS Toolkit and it is really a great tool which helps me in day to day life of the development. 

I have created many lambdas using a serverless framework (https://www.serverless.com/) and I have not created the application using AWS Toolkit or using SAM CLI. 
Basically, I use a serverless-offline plugin in order to debug them locally. 

Can you please tell me how to debug the lambda functions which are created using a serverless framework using AWS toolkit and debug points in IntelliJ?

**Comment by bryceitoc9** [comment]
You may want to reach out to the Serverless framework community for assistance with this since SAM isn't necessarily immediately compatible with the Serverless framework (the "serverless" that makes up the "S" in SAM is unfortunately not a reference to the Serverless framework, which is third-party). I did find this on the Serverless site, would this help? https://www.serverless.com/plugins/serverless-sam

**Comment by sumanth712bs** [comment]
Thank you for the help! :)
